changelog.d/pa-2711.added

```diff
@@ -0,0 +1,2 @@
+Kotlin: Interpolated identifiers in strings, such as "$foo", are now properly
+able to match explicitly interpolated expressions, like "${...}".
```

Comment:
Your test has `...{...}...` (no $), is that just a language equivalence?

Reply:
oops, yeah that's a mistake on my part, good catch